scopes/dependencies/dependency-resolver/dependency-installer.ts

```diff
@@ -164,6 +164,9 @@ export class DependencyInstaller {
             delete linkedDependencies[finalRootDir][manifest.name];
           }
         }
+        if (!packageManagerOptions.hasHarmonyInRootPolicy) {
```

Comment:
Once moved I would rename it.
In the install opts (or in the PM option) context there is no meaning for the has in root policy.
In the options I would make it something like `skipInstallHarmony` / or `doNotInstallHarmony` or `linkTeambitHarmony`. anyway, not nothing with the root policy in the name.
the `hasHarmonyInRootPolicy` name is fine for the dep resolver main runtime API.